packages/core/components/Layout/components/IntroText/IntroText.styles.css

```diff
@@ -0,0 +1,6 @@
+.cove-component__intro-text {
+  padding: 15px;
```

Comment:
can you try using bootstrap utility classes for margin and padding for consistency?